ci/travis_script_manylinux.sh

```diff
@@ -21,9 +21,7 @@
 set -ex
 
 pushd python/manylinux1
-git clone ../../ arrow
-docker build -t arrow-base-x86_64 -f Dockerfile-x86_64 .
-docker run --shm-size=2g --rm -e PYARROW_PARALLEL=3 -v $PWD:/io arrow-base-x86_64 /io/build_arrow.sh
+docker run --shm-size=2g --rm -e PYARROW_PARALLEL=3 -v $PWD:/io -v $PWD/../../:/arrow quay.io/xhochy/arrow_manylinux1_x86_64_base:latest /io/build_arrow.sh
```

Comment:
Does this mean build products will spill in the local checkout?

Reply:
Yes, Python products do.

Reply:
Thanks. That sounds ok to me.

Reply:
How about `ADD`ing arrow to the docker image instead of mounting?

Reply:
We don't run a `docker build` step anymore here and from my experience adding takes quite some time (at least on OSX where the files need to be sent to the docker daemon).

Reply:
Ok, I'll try to add the manylinux image to https://github.com/apache/arrow/pull/2572.
Another question: could We rely cmake's 3rdparty toolchain instead of building the dependencies in additional scripts?

Reply:
We're building them separately as we do some tricks like the boost namespacing but it is also a case of caching. The docker image is pre-built and will only be pulled into the Travis so that the thirdparties are not rebuild every CI run.

Reply:
Perhaps We could define a target to build the 3rdparty deps and cache them similarly. Thanks for the clarification!